### ISSUE TITLE
Exposed API key!!

### ISSUE THREAD

**Issue by Hunter87ff** [debasish-barman-0011/Pinterest-Unsplash-Clone#1]
## What happened?
> There is an api key hardcoded in a file called [script.js](https://github.com/debasish-barman-0011/Pinterest-Unsplash-Clone/blob/main/Js/script.js).

**Comment by debasish-barman-0011** [comment]
Yeah, I noticed that, I'll definitely make it as  an environment variable
in my next git push.
